logsmanagement/README.md

```diff
@@ -0,0 +1,519 @@
+| `bitrate` | The bitrate for the communication, e.g: 9600, 38400, 115200, etc..|
+| `min bytes` | The minimum bytes the serial interface will wait to receive before it begines to process the log message.|
+| `separator` | An optional separator string to determine the end of a log message.|
+| `format` | Specify the format of the incoming data stream. The only option available is 'json'. Note that Format and Separator cannot be used at the same time.|
+
+<a name="custom-charts"/>
+
+## Custom Charts
+
+</a>
+
+In addition to the predefined charts, each log source supports the option to extract 
+user-defined metrics, by matching log records to POSIX Extended Regular Expressions. 
+This can be very useful particularly for `FLB_GENERIC` type log sources, where
+there is no parsing at all by default.
+
+To create a custom chart, the following key-value configuration options must be 
+added to the respective log source configuration section:
+
+    custom 1 chart = identifier
+	custom 1 regex name = kernel
+	custom 1 regex = .*\bkernel\b.*
+	custom 1 ignore case = no
+
+where the value denoted by:
+    - `custom x chart` is the title of the chart.
+    - `custom x regex name` is an optional name for the dimension of this particular metric 
+    (if absent, the regex will be used as the dimension name instead).
+    - `custom x regex` is the POSIX Extended Regular Expression to be used to match log records.
+    - `custom x ignore case` is equivalent to setting `REG_ICASE` when using 
+    POSIX Extended Regular Expressions for case insensitive searches. It is optional and defaults to `yes`. 
+
+`x` must start from number 1 and monotonically increase by 1 every time a new regular expression is configured. 
+If the titles of two or more charts of a certain log source are the same, the dimensions will be grouped together 
+in the same chart, rather than a new chart being created.
+
+Example of configuration for a generic log source collection with custom regex-based parsers:
+
+```
+[Auth.log]
+	enabled = yes
+	update every = 1
+	log type = generic
+	circular buffer max size = 256 # in MiB
+	compression acceleration = 1 # see https://github.com/lz4/lz4/blob/90d68e37093d815e7ea06b0ee3c168cccffc84b8/lib/lz4.h#L195
+	buffer flush to DB = 6 # in sec, default 6 min 4
+	disk space limit = 500 # in MiB, default 500MiB
+	log path = /var/log/auth.log
+
+	custom 1 chart = sudo and su
+	custom 1 regex name = sudo
+	custom 1 regex = \bsudo\b
+	custom 1 ignore case = yes
+
+	custom 2 chart = sudo and su
+	# custom 2 regex name = su
+	custom 2 regex = \bsu\b
+	custom 2 ignore case = yes
+
+	custom 3 chart = sudo or su
+	custom 3 regex name = sudo or su
+	custom 3 regex = \bsudo\b|\bsu\b
+	custom 3 ignore case = yes
+```
+
+And the generated charts based on this configuration:
+
+![Auth.log](https://user-images.githubusercontent.com/5953192/197003292-13cf2285-c614-42a1-ad5a-896370c22883.PNG)
+
+<a name="streaming"/>
+
+## Streaming
+
+</a>
+
+TODO
```

Comment:
@Dim-P probably you missed some information here, or I did not understand  a `TODO` in our documentation.

Reply:
It's something I haven't done yet, thus why marked as `TODO`. Docs are somewhat incomplete yet.